code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -943,6 +943,26 @@
 	smoke.set_up(1, get_turf(P))
 	smoke.start()
 
+/datum/ammo/rocket/frag
+	name = "fragmentation rocket"
```

Comment:
Try to get a sprite for it for the ammo hud.

Reply:
done